scripts/e2e.sh

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env bash
+
+git_ref=$(git rev-parse --short HEAD)
+tower_openshift_project=tower-e2e-"$git_ref"
+
+echo "tower_openshift_project: '"$tower_openshift_project"'" >> e2e_test_config.yml
+echo "tower_openshift_password: '"$TOWER_OPENSHIFT_PASSWORD"'" >> e2e_test_config.yml
+echo "tower_openshift_username: '"$TOWER_OPENSHIFT_USERNAME"'" >> e2e_test_config.yml
+echo "tower_openshift_master_url: '"$OPENSHIFT_MASTER"'" >> e2e_test_config.yml
+
+
+echo "Installing tower on openshift: ${OPENSHIFT_MASTER}"
+ansible-playbook -i ./inventories/hosts \
+playbooks/install_tower.yml \
+--extra-vars "@e2e_test_config.yml"
+
+sleep 120
+#Pass environment variables to config file
+echo "e2e_tower_license: '"$TOWER_LICENSE"'" >> e2e_test_config.yml
+echo "e2e_tower_password: '"$TOWER_PASSWORD"'" >> e2e_test_config.yml
+echo "e2e_tower_username: '"$TOWER_USERNAME"'" >> e2e_test_config.yml
+
+
+echo "Bootstrapping tower in openshift project: $tower_openshift_project"
+ansible-playbook -i ./inventories/hosts \
+playbooks/bootstrap.yml \
+--extra-vars "@e2e_test_config.yml"
+
+oc delete project $tower_openshift_project
+
+#trap "oc delete project $tower_openshift_project"
```

Comment:
This won't delete the namespace on completion if it fails at any point. Did you do something else i cant see to get round this?

If you're using trap, you need to put it at the top of the script before everything else here https://github.com/integr8ly/ansible-tower-configuration/pull/141/files#diff-959b08c8ada2f63d6afa54e0513aa1bcR2:

```
#trap "oc delete project $tower_openshift_project" EXIT
```

Need to verify the exit status is correct on failure though.